### PR TITLE
Rebrand to Freckle

### PR DESCRIPTION
**Pull request by eborden**
:point_up:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,7 +69,7 @@ <h3>Looks like your system can run Front Row, if you have trouble please send us
         <!-- Start of Email Report Section -->
         <div class="row js-send-email-report-div">
           <div class="col-xs-12 text-center">
-            <a class="btn btn-lg btn-info js-frontrow-email-link" href="mailto:support@frontrowed.com?subject=Front Row Connection Test&amp;body=Hi,%0A%0AI did the connection test of Front Row. Please find information about my configuration below.">
+            <a class="btn btn-lg btn-info js-frontrow-email-link" href="mailto:support@frontrowed.com?subject=Freckle Connection Test&amp;body=Hi,%0A%0AI did the connection test of Front Row. Please find information about my configuration below.">
```
Comment (stackptr) [inline index.html]: `body` of the link still mentions Front Row

**Comment by eborden** [comment]
Not sure what we want to do with the zendesk links, but they are scattered all over the place.

**Comment by tfausak** [comment]
😮

**Comment by eborden** [comment]
Hi @tfausak :wave:

**Pull request merged**